app/screens/transactions/Transactions.js

```diff
@@ -2,171 +2,199 @@
 import React, { Component } from 'react';
 import styled from 'styled-components';
 import { connect } from 'react-redux';
-import { Transaction } from '/components/transactions';
-import { Button } from '/basicComponents';
-import { smColors } from '/vars';
-import type { TxList } from '/types';
-import { shell } from 'electron';
+import type { RouterHistory } from 'react-router-dom';
+import { TransactionRow, TransactionsMeta } from '/components/transactions';
+import { CreateNewContact } from '/components/contacts';
+import { Link, WrapperWith2SideBars, SecondaryButton, CorneredWrapper, DropDown } from '/basicComponents';
+import type { TxList, Tx } from '/types';
 import { ScreenErrorBoundary } from '/components/errorHandler';
+import { chevronLeftWhite } from '/assets/images';
+import { smColors } from '/vars';
 
-const Wrapper = styled.div`
-  width: 100%;
-  height: 100%;
-  display: flex;
-  flex: 1;
-  flex-direction: column;
-  padding: 50px 60px;
-`;
-
-const Header = styled.div`
-  font-size: 31px;
-  font-weight: bold;
-  line-height: 42px;
-  color: ${smColors.realBlack};
-  margin-bottom: 20px;
-`;
+const getNumOfCoinsFromTransactions = ({ transactions, isSent }: { transactions: TxList, isSent: boolean }): number => {
```

Comment:
change the function to return all calculated values in single pass instead of calling it multiple times with different params